tests/components/homee/test_config_flow.py

```diff
@@ -0,0 +1,138 @@
+"""Test the Homee config flow."""
+
+from unittest.mock import ANY, MagicMock, patch
+
+from pyHomee import HomeeAuthFailedException, HomeeConnectionFailedException
+import pytest
+
+from homeassistant.components.homee import config_flow
+from homeassistant.components.homee.const import DOMAIN
+from homeassistant.config_entries import SOURCE_USER
+from homeassistant.const import CONF_HOST, CONF_PASSWORD, CONF_USERNAME
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResultType
+
+from .conftest import HOMEE_ID, HOMEE_IP, TESTPASS, TESTUSER
+
+from tests.common import MockConfigEntry
+
+
+async def test_config_flow(
+    hass: HomeAssistant,
+    mock_homee: MagicMock,  # pylint: disable=unused-argument
+    mock_config_entry: MockConfigEntry,  # pylint: disable=unused-argument
+    mock_setup_entry: MagicMock,  # pylint: disable=unused-argument
+) -> None:
+    """Test the complete config flow."""
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": SOURCE_USER}
+    )
+
+    expected = {
+        "data_schema": config_flow.AUTH_SCHEMA,
+        "description_placeholders": None,
+        "errors": {},
+        "flow_id": ANY,
+        "handler": DOMAIN,
+        "step_id": "user",
+        "type": FlowResultType.FORM,
+        "last_step": None,
+        "preview": None,
+    }
+    assert result == expected
+
+    flow_id = result["flow_id"]
+
+    final_result = await hass.config_entries.flow.async_configure(
```

Comment:
```suggestion
    result = await hass.config_entries.flow.async_configure(
```
You can just overwrite result btw since you don't need the other one anyway